faqs/README.md

```diff
@@ -154,7 +154,7 @@ Don't worry, as far as we know, there are no fake Raspberry Pis. The processors
 
 ### I want to be a Raspberry Pi reseller.
 
-We have an exclusive manufacturing and distribution arrangement with [RS](http://uk.rs-online.com/web/generalDisplay.html?id=raspberrypi) and [Farnell](https://www.element14.com/community/community/raspberry-pi). Resellers buy the Raspberry Pi in bulk from them (which reduces shipping costs to nearly nothing) and sell on. You do not need any special licence to resell, and the distributors are very happy to sell on to resellers. If you are interested in joining our [Approved Reseller](https://www.raspberrypi.org/blog/approved-reseller/) programme, contact us by emailing info@raspberrypi.org.
+We have an exclusive manufacturing and distribution arrangement with [RS](https://www.okdo.com/c/pi-shop/the-raspberry-pi/) and [Farnell](https://www.element14.com/community/community/raspberry-pi). Resellers buy the Raspberry Pi in bulk from them (which reduces shipping costs to nearly nothing) and sell on. You do not need any special licence to resell, and the distributors are very happy to sell on to resellers. If you are interested in joining our [Approved Reseller](https://www.raspberrypi.org/blog/approved-reseller/) programme, contact us by emailing info@raspberrypi.org.
```

Comment:
I've got no idea how RS and OKDO interrelate, but this URL stays on the RS website (without redirecting to OKDO) : https://uk.rs-online.com/web/b/raspberry-pi/

Reply:
This is probably a good question for Samantha Snyder, our new hire?